src/Microsoft.Restier.Core/ApiConfiguration.cs

```diff
@@ -120,5 +100,51 @@ internal TaskCompletionSource<IEdmModel> CompeteModelGeneration(out Task<IEdmMod
             running = null;
             return source;
         }
+
+        public sealed class Customizer
+        {
+            public Customizer()
+            {
+                InnerMost = new Anchor();
+                OuterMost = new Anchor();
+                Overrides = new Anchor();
+                PrivateApi = new Anchor();
+            }
```

Comment:
Maybe remove Overrides
And we have 
InnerServiceChain (tail to service registered by RESTier code)
OutServiceChain (Head to the service registered by RESTier code)
InternalServiceChain (For RESTier code to use only).